tests/foreman/ui/test_subnet.py

```diff
@@ -49,9 +49,7 @@ def test_create_subnet_with_different_names(self):
                         subnet_network=gen_ipaddr(ip3=True),
                         subnet_mask=gen_netmask(),
                     )
-                    self.assertIsNotNone(
-                        self.subnet.search_subnet(subnet_name=name)
-                    )
+                    self.assertIsNotNone(self.subnet.search(name))
```

Comment:
Here and till the end of module there was originally `search_and_validate`, not just regular `search` func. Was that changed on purpose?